### ISSUE TITLE
route53: Missing Waiters that were in v1 SDK

### ISSUE THREAD

**Issue by qhenkart** [aws/aws-sdk-go-v2#1148]
Confirm by changing [ ] to [x] below to ensure that it's a bug:
- [x] I've gone though the [API reference](https://docs.aws.amazon.com/sdk-for-go/v2/api/)
- [x] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-go) for answers
- [x] I've searched for [previous similar issues](https://github.com/aws/aws-sdk-go-v2/issues) and didn't find any solution
  
**Describe the bug**
The route53 package in v2 has no waiters, [however the package in v1 does](https://docs.aws.amazon.com/sdk-for-go/api/service/route53/#Route53.WaitUntilResourceRecordSetsChanged)

**Version of AWS SDK for Go?**
1.2

**Version of Go (`go version`)?**
1.16

**To Reproduce (observed behavior)**
https://docs.aws.amazon.com/sdk-for-go/api/service/route53/#Route53.WaitUntilResourceRecordSetsChanged

https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/route53#Client

**Expected behavior**
The waiter to exist in v2 like it did in v1

**Additional context**
basically the same issue as #1111 but a different package

**Comment by jasdel** [comment]
Thanks for reporting this issue @qhenkart. The v2 SDK did not pull over all modeled waiters from the v1 SDK. The AWS Route53 waiters are an example of some of the waiters that have not been ported yet. These waiters were not ported due to a change in how the APIs model their features, and waiters like Route53's have not been added yet.

We'll use this GitHub issue to track this waiter for Route53.

**Comment by skmcgrail** [comment]
Waiter was added back in `v.1.4.0` of the route53 module. https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/route53#NewResourceRecordSetsChangedWaiter
